vsm/src/vespa/vsm/vsm/vsm-adapter.h

```diff
@@ -97,17 +97,12 @@ typedef std::shared_ptr<DocsumTools> DocsumToolsPtr;
 class VSMConfigSnapshot {
 private:
     const vespalib::string _configId;
-    const config::ConfigSnapshot _snapshot;
+    std::unique_ptr<config::ConfigSnapshot> _snapshot;
```

Comment:
Should this be `unique_ptr<const config::ConfigSnapshot>` to match the previous constness?

Reply:
Fixed